bin/lib/check_reqs.js

```diff
@@ -78,21 +79,46 @@ module.exports.check_ant = function() {
     });
 };
 
+module.exports.get_gradle_wrapper = function() {
+    var androidStudioPath;
+    if(os.platform() == "darwin") {
```

Comment:
@dpogue Honestly, we're just installing the wrapper, so any will work.  I wanted to delete https://github.com/apache/cordova-android/blob/master/bin/templates/cordova/lib/builders/GradleBuilder.js#L187 for a while, since it replaces everything with an old version of Gradle that just gets updated anyway, but we might just have to keep it in and update it to the latest Gradle supported.

FYI: My homebrew keeps fetching 2.2, and Android Studio 2.3 is packaged with 3.2 on my Windows and Mac partitions.